spec/rails_helper.rb

```diff
@@ -156,14 +156,14 @@
   # config.filter_gems_from_backtrace("gem name")
 
   # Explicitly set a seed and time to ensure deterministic Percy snapshots.
-  config.around(:each, percy: true) do |example|
-    Timecop.freeze("2020-05-13T10:00:00Z")
-    prev_random_seed = Faker::Config.random
-    Faker::Config.random = Random.new(42)
+  # config.around(:each, percy: true) do |example|
+  #   Timecop.freeze("2020-05-13T10:00:00Z")
+  #   prev_random_seed = Faker::Config.random
+  #   Faker::Config.random = Random.new(42)
 
-    example.run
+  #   example.run
 
-    Faker::Config.random = prev_random_seed
-    Timecop.return
-  end
+  #   Faker::Config.random = prev_random_seed
+  #   Timecop.return
+  # end
```

Comment:
If we're going to comment this out, we should probably also just remove the PERCY_TOKEN from the travis env vars.

Reply:
I left it bc I figured we would want to try to fix it at some point.

Reply:
(Otherwise Percy is going to be getting a bunch of wonky snapshots without this setting)

Reply:
@mstruve Sorry, to clarify: I mean that we should temporarily remove the PERCY_TOKEN from Travis settings until we can figure out how to fixed, which will effectively disable Percy until we can fix this.